meson.build

```diff
@@ -179,6 +179,19 @@ endif
 #===============================================================================
 # Install Configuration
 #===============================================================================
+if get_option('all-in-one')
+    cp = find_program('cp', required: true)
+    custom_target('copy script',
+        input : 'scripts/generate-all-in-one.lua',
+        output : 'generate-all-in-one.lua',
+        command : [cp, '@INPUT@', '@OUTPUT@'],
+        install : false,
+        build_by_default : true
+    )
+    run_command(cp, '-r', 'data', meson.current_build_dir())
```

Comment:
I just passed in the binary verison; we've been talking about having this available for a long while anwyay; may as well do it here; seems like path of least resistance.

Reply:
Please don't do this. This will destroy compatibility with Windows. You can add a `meson.build` file to configure the file correctly.

Reply:
My suggestion is definitely rewriting the AIO script in Python. I have [something similar](https://github.com/lite-xl/playground/blob/main-v2/scripts/generate-wasm-bundle.py) in playground that copies the file into the build directory and calls `file_packager` (in which case you can just generate C code). This is more cross-platform than requiring `sh`, `find` and `cp`, or lua, since Python is installed in all our CI and meson kinda requires you to have one installed too.

Reply:
I think actually the way to go is with a simple C program; it's likely most compatible all around. That's how I had it originally, and apparently meson fully supports creating an intermediate executable as part of the build process.

Reply:
That's fine by me. Do not that this will still be fairly annoying to do in C, especially with reading directories the cross-platform way. I've been here and done that, which is why I recommended the Python script. It may or may not possible to just not touch the output if the file hadn't changed at all, and hopefully meson will not regenerate everything and link Lite XL. I really have a bad experience with relinking after working with Playground (the linking process took 10 seconds)